content/en/docs/setup/_index.md

```diff
@@ -4,11 +4,97 @@ reviewers:
 - erictune
 - mikedanese
 no_issue: true
-weight: 30
 title: Setup
 main_menu: true
+weight: 30
+content_template: templates/concept
 ---
+{% raw  %}
+
+{{% capture overview %}}
+
+You can run Kubernetes almost anywhere, from your laptop to VMs on a cloud
+provider to a rack of bare metal servers. Deciding where to run Kubernetes
+depends on what resources you have available and how much flexibility you need.
+
+Set up a fully-managed cluster by running a single command or craft your own
+customized cluster on your bare metal servers. Use this page to find the type
+of solution that best fits your needs.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Local-machine Solutions
+
+A local-machine solution is a perfect way to get started with Kubernetes. You
+can create and test Kubernetes clusters without worrying about consuming cloud
+resources and quotas.
+
+You should pick a local solution if you want to:
+
+* Try or start learning about Kubernetes
+* Develop and test clusters locally
+
+Pick a [local-machine solution](/docs/setup/pick-right-solution/#local-machine-solutions).
+
+## Hosted Solutions
+
+Hosted solutions are the easiest way to create and maintain Kubernetes clusters. They completely 
```

Comment:
In this case, maybe "convenient" instead of "easiest."

In the sentence, "They **completely** manage and operate your clusters so you don’t have to," I'm not sure if *completely* is accurate. You still have to configure and create the clusters, so there is some management involved, but the manage overhead is significantly lower.